src/formatters/resultFormatter.ts

```diff
@@ -50,6 +51,15 @@ export abstract class ResultFormatter {
     });
   }
 
+  protected sortTestResults(results: Failures[] | Successes[] = []): Failures[] | Successes[] {
+    return results.sort((a: Successes, b: Successes) => {
```

Comment:
The typing here is a bit wonky and I'm surprised TS doesn't complain.  Assuming SDR doesn't provide a common base type for `Failure` and `Success`, it would be better to have 1 type here that only used `name`, and `methodName` properties.